### ISSUE TITLE
Any update on the release?

### ISSUE THREAD

**Issue by f-lng** [ezelikman/Context-Is-Everything#2]
Hello Eric,

Is there any update on the planned release of this repo? Looking forward to it!

Regards,
Fabian

**Comment by JDvorak** [comment]
I don't think it will ever be released. Presumably, the behavior of the code does not match what was reported by the paper, or the author is embarrassed by the code quality.

**Comment by ezelikman** [comment]
Sorry for the delay! The intent was to focus on conferences. More parts will be published as they're cleaned up (e.g. the Erdos generation, an implementation of the iterative search, etc)

**Comment by jiqiujia** [comment]
so is everything released? The descriptions in paper are very confusing. I only get to understand how to calculate the sentence embedding through the pseudo code.
